packages/next/taskfile.js

```diff
@@ -698,6 +698,21 @@ export async function ncc_amphtml_validator(task, opts) {
     .source(relative(__dirname, require.resolve('amphtml-validator')))
     .ncc({ packageName: 'amphtml-validator', externals })
     .target('src/compiled/amphtml-validator')
+
+  const validatorRes = await fetch(
+    'https://cdn.ampproject.org/v0/validator_wasm.js'
```

Comment:
Is that a permalink or does it automatically get new versions? It should be an exact version so that PRs don't start to randomly fail because the vendored source changes.

Reply:
This is the exact URL used by the library https://github.com/ampproject/amphtml/blob/c8bbc75e4274d7ef37d0b218344cfa937020b62b/validator/js/nodejs/index.js#L289-L290

Reply:
We could move it after ncc'ing step so it can change without invalidating the vendored

Reply:
updated

Reply:
I suspect it still is some form of "get me the latest source" which isn't a problem if you keep it in memory like they do. Makes CI non-reproducible but I also don't have a better suggestion other than moving the vendoring to an ad-hoc task. But that's not compatible with how we clean so 🤷

Reply:
I suspect it still is some form of "get me the latest source" which isn't a problem if you keep it in memory like they do. Makes CI non-reproducible but I also don't have a better suggestion other than moving the vendoring to an ad-hoc task. But that's not compatible with how we clean so 🤷 

We definitely have an issue with flaky tests so this is a good step in the right direction.